jablib/src/main/java/org/jabref/logic/openoffice/style/JStyle.java

```diff
@@ -277,19 +276,19 @@ public void ensureUpToDate() throws IOException {
      */
     private void reload() throws IOException {
         if (styleFile != null) {
-            Path resourcePath = styleFile;
             if (fromResource) {
-                try {
-                    URL resUrl = JStyle.class.getResource(path);
-                    if (resUrl != null) {
-                        resourcePath = Path.of(resUrl.toURI());
+                URL resUrl = JStyle.class.getResource(path);
+                if (resUrl != null) {
+                    try (InputStream stream = resUrl.openStream()) {
```

Comment:
I was thinking of `...getResourceAsStream`, but I had no time to experiment. Maybe at a follow-up?

We need `getResourceAsStream` for GraalVM - see https://github.com/JabRef/jabref-koppor/pull/693